react/tic-tac-toe/src/app/components/Topbar/index.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/button-has-type */
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import actionsCreators from '../../../redux/Login/actions';
+import userLogo from '../../assets/user-logo.svg';
+
+import styles from './styles.module.scss';
+
+const Topbar = props => {
```

Comment:
As a convention we follow, stateless components should be defined as regular functions, not arrow.